Revamped Overview Articles (#1131)
telerik/blazor-docs: components/chart/overview.md

> 

Reviewer: Each Overview page should have a titled section for all features that are explained on a separate dedicated page. If the features are too many, you can [group them in one section or enumerate them in a list](https://github.com/telerik/blazor/wiki/Docs-Overview-Page-Template). See the [Grid Overview](https://docs.telerik.com/blazor-ui/components/grid/overview) and sections "Data Operations", "Column Features" and "More Grid Features".

The <a href="https://www.telerik.com/blazor-ui/chart" target="_blank">Blazor Chart component</a> allows you to visualize data to your users in a meaningful way so they can draw conclusions. You can use a variety of chart types and control all aspects of the chart's appearance - from colors and fonts, to paddings, margins and templates.

> ## Creating Chart

Reviewer: https://github.com/telerik/blazor/wiki/Docs-Overview-Page-Template

```suggestion
## Creating Blazor Chart
```

}
````

## Chart Reference and Methods

To execute Chart methods, obtain reference to the component instance via `@ref`.

 @[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Method  | Description |
|---------|-------------|

> | Refresh | You can use that method to programmatically re-render the component.    |

Reviewer: ```suggestion
| Refresh | Use the method to programmatically re-render the Chart.  |
```

}
````

> ## Component Reference and Methods

Reviewer: ```suggestion
## Chart Reference and Methods
```
